### DISCUSSION TITLE
Cast Shadows: Shadows only. Not working.

### DISCUSSION THREAD

**Discussion by frostisenjostein** [visualdesigncafe/nature-renderer#92] [Help]
I'm trying to use shadow proxies set to only cast shadows. This works well as prefabs, and when placed with the unity terrain system, but as soon as I enable Nature Renderer for rendering, it shows the shadow proxy renderer as well as the main model. This gives a duplicate mesh.

**Comment by frostisenjostein** [comment]
This is Unity 6, HDRP

**Comment by maxplooi** [comment]
Thank you for reporting this, I have fixed the issue in version **6.0.1**. You can download the latest version via the Asset Store.
